### ISSUE TITLE
Region of interest analysis

### ISSUE THREAD

**Issue by rob-luke** [mne-tools/mne-nirs#101]
Currently statistics are computed per channel. Need to implement a way to collapse multiple channels to a region of interest analysis.

The computation/statistics are described in equations 36-37 in section 6.3 of [Santosa, Hendrik, et al. "The NIRS brain AnalyzIR toolbox." Algorithms 11.5 (2018): 73](https://www.mdpi.com/1999-4893/11/5/73/htm).

**Comment by rob-luke** [comment]
Regions of interest should be specified in the same manner as in [MNE plot image](https://mne.tools/dev/generated/mne.BaseEpochs.html?highlight=plot_image#mne.BaseEpochs.plot_image)...

```python
group_by=dict(Left_ROI=[1, 2, 3, 4], Right_ROI=[5, 6, 7, 8])
```

**Comment by rob-luke** [comment]
Closed in #137
